x/fswap/spec/01_concepts.md

```diff
@@ -0,0 +1,60 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Swap
+
+
+The `x/fswap` module defines a `Swap` type in which a coin is allowed to be swapped into another coin on the chain.
+You could find detailed information in the [Protobuf reference](../../../proto/lbm/fswap/v1/fswap.proto#L9-L16) 
+
+```go
+type Swap struct {
+  FromDenom             string
+  ToDenom               string
+  AmountCapForToDenom   sdk.Int
+  SwapRate              sdk.Dec
+}
+```
+
+Anyone could use one of the following two transcations to swap `FromDedenom` to `ToDenom`.
+1. `simd tx fswap swap [from] [from_coin_amount] [to_denom]`
+    - this transcation could swap a specified amount of `from_denom` via [`MsgSwap`](../../../proto/lbm/fswap/v1/tx.proto#L17-L24)
+2. `simd tx fswap swap-all [from_address] [from_denom] [to_denom]`
+    - this transcation could swap all of `from_denom` under `from_address` via [`MsgSwapAll`](../../../proto/lbm/fswap/v1/tx.proto#L28-L33)
+
+When the swap is triggered, the following event will occur:
+1. `from_denom` will be sent from `from_address` to `x/fswap` module
+2. `x/fswap` module will burn `from_denom`
+3. `x/fswap` module will mint `to_denom` of equivalent value
+4. these `to_denom` will sent to `from_address`
+5. `EventSwapCoins` will be emitted
+
+## Config
+
+The `x/fswap` module defines a `Config` type for managing the maximum number of Swaps allowed on chain  through `MaxSwaps`. Additionally, `UpdateAllowed` specifies whether `Swap` can be modified.
+
+```go
+type Config struct {
+	MaxSwaps      int
+	UpdateAllowed bool
+}
+```
+
+## MsgSetSwap
+
+`MsgSetSwap` is not a proposal; it is a regular message. Other modules can include `MsgSetSwap` in their proposals to set `Swap`. If the proposal passes, the `Swap` can be used on chain.
```

Comment:
It is a little bit confusing, It says `MsgSetSwap is not proposal` then `If the proposal passes`.
How about this `MsgSetSwap is a message-based proposal`?

Reply:
[As @0Tech said ](https://github.com/Finschia/finschia-sdk/pull/1419#discussion_r1637493076), MsgSetSwap is not a proposal. 
I think `the proposal passes` here refers to `Other modules can include MsgSetSwap in their proposals` mentioned above.